conduct/artifacts/CODE_OF_CONDUCT_PREFIX.md

```diff
@@ -0,0 +1,44 @@
+# Code of Conduct
+
+The OpenJS Foundation and its member projects use Contributor Covenant as their code of conduct. The full text is included below in English, and [translations](https://www.contributor-covenant.org/translations) are available on the Contributor Covenant website.
```

Comment:
this should reflect that we use a modified version of the Contributor Covenant and that the translations do not reflect those modifications.

Reply:
I've applied the recommended changes. Thank you for the feedback.